Android/app/src/main/java/io/github/project_travel_mate/utilities/ContributorsFragment.java

```diff
@@ -0,0 +1,184 @@
+package io.github.project_travel_mate.utilities;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.preference.PreferenceManager;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.GridView;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+import adapters.ContributorsAdapter;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import io.github.project_travel_mate.R;
+import objects.Contributor;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import utils.ExpandableHeightGridView;
+
+import static utils.Constants.API_LINK_V2;
+import static utils.Constants.USER_TOKEN;
+
+/**
+ * @author amoraitis
+ * A simple {@link Fragment} subclass.
+ * Use the {@link ContributorsFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class ContributorsFragment extends Fragment {
+
+    @BindView(R.id.android_contributors_gv)
+    ExpandableHeightGridView android_contributors_gv;
+    @BindView(R.id.server_contributors_gv)
+    ExpandableHeightGridView server_contributors_gv;
+    private String mToken;
+    private Activity mActivity;
```

Comment:
why do we need to hold a ref to Activity? since we can access the parent activity from a fragment via getActivity()

Reply:
I will reference this in the next commit, also this was project's fragment template